rules/windows/persistence_ms_outlook_vba_template.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = "Detects attempts to establish persistence on an endpoint by installing a rogue Microsoft Outlook VBA Template."
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Persistence via Microsoft Outlook VBA"
+references = [
+    "https://www.mdsec.co.uk/2020/11/a-fresh-outlook-on-mail-based-persistence/",
+    "https://www.linkedin.com/pulse/outlook-backdoor-using-vba-samir-b-/",
+]
+risk_score = 43
+rule_id = "397945f3-d39a-4e6f-8bcb-9656c2031438"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
```

Comment:
Was the main reason for eql here to not have to escape the paths when using KQL wildcarding?

Reply:
wildcard and case insensitivity support in the future with wildcard